src/components/MenuItem.js

```diff
@@ -65,7 +70,7 @@ const MenuItem = ({
                 </View>
                 {shouldShowRightArrow && (
                     <View style={styles.createMenuIcon}>
-                        <Icon src={ArrowRight} fill={getIconFillColor(getButtonState(hovered, pressed))} />
+                        <Icon src={iconRight} fill={getIconFillColor(getButtonState(hovered, pressed))} />
```

Comment:
`shouldShowRightArrow` is a bit confusing now, this should be updated to `shouldShowRightIcon`